scripts/connectors/media-wiki-RecentChange-event.schema.json.backup

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
I assume that this file is a documentation/snapshot-of-schema artifact?

Reply:
@javabrett that's exactly right.  It's insurance, a snapshot, but not used by the code.  Is there a suggested naming convention that would identify it as such.

Reply:
I think files named `.backup` or similar just trigger my review senses ... it's fine as-is, just checking really - it could probably just not have `.backup` although I appreciate that is attempting to hint that this file isn't used in processing.  All I can come up with is `media-wiki-RecentChange-event.reference.schema.json`.

Reply:
@javabrett latest commit renames the file to `media-wiki-RecentChange-event.reference.schema.json`